lib/triggerParser.js

```diff
@@ -31,15 +31,18 @@ var EXIT = function() {
         EXIT
       );
       return;
-    } else if (/Firebase config variables are not available/.test(e.message)) {
+    }
+    if (/Firebase config variables are not available/.test(e.message)) {
       process.send(
         {
           error:
-            'Error occurred while parsing your function triggers. Please ensure you have the latest firebase-functions SDK by running "npm i --save firebase-functions@latest" inside your functions folder.\n\n' +
+            "Error occurred while parsing your function triggers. " +
```

Comment:
(Very nit-picky, feel free to ignore) Should your new piece to the error message also use single quotes instead of double quotes?